index.js

```diff
@@ -10,19 +11,30 @@ import theme from './src/theme'
 
 class ZooniverseMobile extends Component {
   render() {
+    
     return (
       <View style={styles.container}>
-        <StatusBar
-          barStyle="light-content"
-          hidden={false}
-          translucent={false}
-        />
+        <PlatformSpecificStatusBar />
```

Comment:
Is there a reason this component only renders for iOS?

Reply:
Not sure, it was basically the only difference between the two index files before I consolidated them.

Reply:
Ah, gotcha. Didn't notice this used to be the iOS index.